src/modals/CssSnippetFuzzySuggestModal.ts

```diff
@@ -46,10 +61,24 @@ export class CssSnippetFuzzySuggestModal extends FuzzySuggestModal<
 				command: Platform.isMacOS ? "⌘ del" : "ctrl del",
 				purpose: "to delete",
 			},
+			{ command: "tab", purpose: "to toggle state" },
 			{ command: "esc", purpose: "to dismiss" },
 		]);
 	}
 
+    getOrToggleSnippet(item:String, toggleValue : Boolean = false) : Boolean | undefined {
```

Comment:
This is a nitpick, but I'd prefer this to be split out into two separate functions, `getIsSnippetEnabled` and `toggleSnippetEnabled`. `getIsSnippetEnabled` will just check to see if a snippet is enabled and return a boolean. `toggleSnippetEnabled` will toggle whether the snippet is enabled or not and will not return anything.

I won't block this if you don't change it, just a preference I have.

Reply:
I'm usually doing this is two functions myself, but due to the snippet name manipulation needing to be repeated in n both of those command I kind of felt it was a little more natural to avoid the duplicated code. But can do it, if you still like that better.

Reply:
You could still avoid the duplicated code by calling `getIsSnippetEnabled` from the `toggleSnippetEnabled` function

Reply:
There is a slight difference related to removal of the file extension once or twice, but it do look clearer with different methods.